lib/Interpreter/InterOp.cpp

```diff
@@ -1829,6 +1830,21 @@ namespace InterOp {
     return 0;
   }
 
+  TInterp_t CreateInterpreter(const char *resource_dir) {
+    std::string MainExecutableName =
+      llvm::sys::fs::getMainExecutable(nullptr, nullptr);
```

Comment:
We should lift some of the infrastructure I mentioned here https://github.com/compiler-research/InterOp/blob/41a32cce4af78c3b1225b90c84515f144055337f/unittests/InterOp/Utils.cpp#L21-L62

and check if no `resource_dir` is passed we fall back to it.